packages/lib/src/dimension-mapper/DimensionMapper.module.css

```diff
@@ -2,9 +2,8 @@
   display: flex;
   flex-direction: column;
   padding: 1rem 0.375rem 1rem 0.375rem;
-  background-color: var(--primary-light-bg);
+  background-color: var(--h5w-dimMapper--bgColor, ghostwhite);
   overflow: hidden auto;
-  font-size: 0.875em;
```

Comment:
This was making the switch to relative font sizes in `SlicingSlider.module.css` a bit tricky and led to awkward `em` values. I move it to `axisMapper` so I can adjust the font size of the slider elements more granularly.